astropy/wcs/tests/test_wcs.py

```diff
@@ -1183,3 +1183,14 @@ def test_footprint_contains():
 
     hasCoord = test_wcs.footprint_contains(SkyCoord(24,2,unit='deg'))
     assert hasCoord == False
+
+
+def test_cunit():
+    # Initialsinig WCS 
+    w1 = wcs.WCS(naxis=2)
+    # Initialising the values of cunit   
+    w1.wcs.cunit = ['deg', 'm/s']
+
+    assert w1.wcs.cunit == w1.wcs.cunit
```

Comment:
Could you make another WCS with different units and then make sure that ``!=`` works as expected too when the units are different?

Reply:
Could you also test for equality on different instances? This could just happen to work because both reference the same `cunit`. For example a `w1.wcs.cunit == w3.wcs.cunit` where `w3.wcs.cunit = ['deg', 'm/s']`

Reply:
Not sure if the coverage report is wrong but it reports the function as not covered by the tests. Could you make sure that the test is tested and the new richcmp function is actually called?

Reply:
I tested that  new richcmp func is actually called not sure about  coverage report.

Reply:
Ah, I just noticed that the flake check failed. Maybe that's the reason the coverage isn't picked up.